frontend/src/concepts/pipelines/content/experiment/CreateExperimentButton.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { ExperimentKFv2 } from '~/concepts/pipelines/kfTypes';
+import ManageExperimentModal from '~/concepts/pipelines/content/experiment/ManageExperimentModal';
+
+type CreateExperimentButtonProps = {
+  onCreate?: (experiment: ExperimentKFv2) => void;
+} & Omit<React.ComponentProps<typeof Button>, 'onClick'>;
+
+const CreateExperimentButton: React.FC<CreateExperimentButtonProps> = ({
+  onCreate,
+  children,
+  ...buttonProps
+}) => {
+  const { apiAvailable, refreshAllAPI, pipelinesServer } = usePipelinesAPI();
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <>
+      <Button
+        {...buttonProps}
+        isDisabled={!apiAvailable || buttonProps.isDisabled || !pipelinesServer.compatible}
+        onClick={() => setOpen(true)}
+      >
+        {children || 'Create experiment'}
+      </Button>
+      <ManageExperimentModal
```

Comment:
I know you didn't write this component, but I feel like this is a good opportunity to rename it to `CreateExperimentModal`. I'm not sure what other "management" is happening to experiments, as I see "Create" is hardcoded inside of it already.